lib/features/modeling/behavior/SubProcessPlaneBehavior.js

```diff
@@ -411,6 +411,15 @@ SubProcessPlaneBehavior.prototype._moveChildrenToShape = function(source, target
     return;
   }
 
+  // add external labels that weren't children of sub process
+  children = children.concat(children.reduce((labels, child) => {
```

Comment:
PhantomJS does not approve this closure. :wilted_flower:

Reply:
Argh!